### DISCUSSION TITLE
How to bypass google 6 minutes script running

### DISCUSSION THREAD

**Discussion by lstbrian** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#34] [Ideas]
Jan 25, 2022, 12:22:55 AM	Error	Exceeded maximum execution time
MAX 6 minutes to run a script

How to bypass it?

**Comment by lstbrian** [comment]
https://inclucat.wordpress.com/2021/12/14/an-easy-way-to-deal-with-google-apps-scripts-6-minute-limit/

Do you think you can include LongRun class so the script will not fail after 6 minutes?

**Comment by chriskyfung** [comment]
@lstbrian Thanks for your information!

It maybe a solution for the limitation, I can try it in the future.
